taxonomies/categories.txt

```diff
@@ -47815,6 +47815,10 @@ ciqual_food_code:en:39511
 ciqual_food_name:en:Custard dessert, refrigerated or canned (average)
 ciqual_food_name:fr:Crème dessert, rayon frais ou appertisée (aliment moyen)
 
+#Those puddings seem to be sold only in Spain and Portugal, no need to translate it
+<en:Creamy puddings
+en:Natillas con galleta
```

Comment:
If it's only sold in es/pt why not put the entry in es/pt ?
```suggestion
#Those puddings seem to be sold only in Spain and Portugal, no need to translate it
<en:Creamy puddings
es:Natillas con galleta
```

Reply:
That was a typo, I corrected it, thak you!